modules/checklist/assets/js/components/checklist-card-content.js

```diff
@@ -17,8 +33,16 @@ const ChecklistCardContent = ( props ) => {
 				</Typography>
 			</CardContent>
 			<CardActions>
-				<Button size="small" color="secondary" variant="text" className="mark-as-done">{ __( 'Mark as done', 'elementor' ) }</Button>
-				<Button size="small" variant="contained" className="cta-button">{ CTA }</Button>
+
```

Comment:
Why do we remove buttons from component?  Maybe we should render mark as done conditionally, based on isLocked?

Reply:
It wasn't removed, just arranged differently for readability

Reply:
@ronenelementor I see 1 button in cards actions, instead of 2

Reply:
NICE CATCH!